### PR TITLE
Dashboard views have Aspects section

### LINKED ISSUE #1940: Aspects preview section on Dashboard tab Hub + Challenge
## Description
As a user I want to be able to see what Aspects are available

## Acceptance criteria

- [x] A aspects preview box under the Hub Dashboard tab 
- [x] A aspects preview box under the Challenge Dashboard tab 
- [x] A aspects preview box under the Opportunities Dashboard tab 
- [x] Aspects preview box has activity statistics 
- [x] Extend API -> activity to have aspects count on hub, challenge, opportunity level
- [x] We show 2 cards
- [x] the aspects preview is below challenges / opportunities / projects

## Additional Context

https://xd.adobe.com/view/e2d9a902-8f26-44bc-bdc8-58d1ca9cfa1e-b78a/

### PR DESCRIPTION
**Pull request by me-andre**
### Describe the background of your pull request

Resolves #1940 

### Additional context

Add any other context

### Governance 

- [ ] Documentation is added
- [ ] Test cases are added or updated

By submitting this pull request I confirm that:
- I've read the contributing document https://github.com/alkem-io/.github/blob/master/CONTRIBUTING.md.
- I've read and understand the Code of Conduct https://github.com/alkem-io/.github/blob/master/CODE_OF_CONDUCT.md.
- I understand that any contributions or suggestions I made may make it into the actual code. I've read the License 
     https://github.com/alkem-io/Coordination/blob/master/LICENSE.
- I understand that submitting the pull request requires agreeing to and signing the contributor license agreement.

### CODE REVIEW

File: src/components/composite/aspect/DashboardSectionAspects/DashboardSectionAspects.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  aspectsCount?: number;
+  hubNameId?: string;
+  challengeNameId?: string;
+  opportunityNameId?: string;
+}
+
+const DashboardSectionAspects: FC<DashboardSectionAspectsProps> = ({ aspects, aspectsCount, ...parentEntityIds }) => {
+  const { t } = useTranslation();
+
+  const headerText = aspectsCount ? `${t('common.aspects')} (${aspectsCount})` : t('common.aspects');
```
Comment (hero101) [inline src/components/composite/aspect/DashboardSectionAspects/DashboardSectionAspects.tsx:18]: this won't display `0`.

File: src/components/composite/sections/ContextSection.tsx [hunk 2]
```diff
@@ -2,16 +2,18 @@ import MenuBookIcon from '@mui/icons-material/MenuBook';
 import PeopleAltIcon from '@mui/icons-material/PeopleAlt';
```
Comment (hero101) [inline src/components/composite/sections/ContextSection.tsx:2]: How is this change related to the story?

File: src/components/composite/sections/ContextSectionIcon.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React, { ComponentType, FC } from 'react';
```
Comment (hero101) [inline src/components/composite/sections/ContextSectionIcon.tsx:1]: How is this change related to the story?

**Review by hero101 — CHANGES_REQUESTED** [review]

File: src/components/composite/aspect/DashboardSectionAspects/DashboardSectionAspects.tsx (hunk 1 above)
Comment (me-andre) [reply src/components/composite/aspect/DashboardSectionAspects/DashboardSectionAspects.tsx:18]: Well, this is intentional.
Should we display `Aspects (0)` on an empty Aspects card?

File: src/components/composite/sections/ContextSection.tsx (hunk 2 above)
Comment (me-andre) [reply src/components/composite/sections/ContextSection.tsx:2]: It's not, but it was initially. Most of the work I did before Aspects were moved from Context to Dashboard. Should I revert changes to the `ContextSection`? (I mean, they look as an improvement to me).

File: src/components/composite/aspect/DashboardSectionAspects/DashboardSectionAspects.tsx (hunk 1 above)
Comment (hero101) [reply src/components/composite/aspect/DashboardSectionAspects/DashboardSectionAspects.tsx:18]: We should have some indication that there are no aspects

File: src/components/composite/sections/ContextSectionIcon.tsx (hunk 3 above)
Comment (me-andre) [reply src/components/composite/sections/ContextSectionIcon.tsx:1]: Please see https://github.com/alkem-io/client-web/pull/1973#discussion_r849360235

File: src/components/composite/sections/ContextSection.tsx (hunk 2 above)
Comment (hero101) [reply src/components/composite/sections/ContextSection.tsx:2]: They are an improvement - don't revert them, but also-  where do you draw the line?

File: src/components/composite/aspect/DashboardSectionAspects/DashboardSectionAspects.tsx (hunk 1 above)
Comment (me-andre) [reply src/components/composite/aspect/DashboardSectionAspects/DashboardSectionAspects.tsx:18]: I'm used to having a thing like "no items" view in place of a list / table when it's empty. If we have such a thing, I can use it there. But of course I can also make a quick change and just show `Aspects (0)`. Should I stick with the latter?

Comment (hero101) [reply src/components/composite/aspect/DashboardSectionAspects/DashboardSectionAspects.tsx:18]: Yes, just display 0 for now.

Comment (me-andre) [reply src/components/composite/aspect/DashboardSectionAspects/DashboardSectionAspects.tsx]: Looks good?

Comment (hero101) [reply src/components/composite/aspect/DashboardSectionAspects/DashboardSectionAspects.tsx]: Just from looking at the code, it seems to be that it won't show 0 on no aspects. It's either the label or label + count

**Review by hero101 — APPROVED** [review]
